build/pipelines/templates/extensions-ci.yml

```diff
@@ -6,6 +6,9 @@ parameters:
   displayName: Run Extension Tests?
   type: boolean
   default: false
+- name: Solution
+  type: string
+  default: not-specified
```

Comment:
Will this continue to work for all the other builds?

Reply:
The condition doesn't seem to be evaluating this properly

Reply:
checking this

Reply:
Thank you for catching this. Started the PR for fix here - https://github.com/Azure/azure-functions-dotnet-worker/pull/2032